### DISCUSSION TITLE
Running with flag --eraseCache and --forceUpgrade?

### DISCUSSION THREAD

**Discussion by finicu212** [itzg/docker-minecraft-server#1857] [Q&A] [answered]
Hi, new hoster here, ran into this issue: https://bugs.mojang.com/browse/MC-142134. To fix, I need to restart the server with --eraseCache and --forceUpgrade, but how do I run the .jar with these flags? I can't seem to find any env vars to run with -e flag that would set these to true.

Thanks!

**Comment by itzg** [comment, marked as answer]
There a few ways (depending on how you're running the container) but this works in all cases 

https://github.com/itzg/docker-minecraft-server#jarfile-options
